man/osg-incommon-cert-request.1

```diff
@@ -44,6 +44,9 @@ May be specified more than once for additional SANs.
 .TP
 .BR \-d, \-\-directory
 The directory to write the host certificate(s) and key(s).
+.TP
+.BR \-O, \-\-orgcode
+Specify alternative organization and department codes for the InCommon Certificate Service. Defaults are Fermilab\'s codes.
```

Comment:
Can you describe how to find these codes in the InCommon web interface?  If so it would be good to add that here.

Reply:
Added, please review.